zebra-chain/src/transaction/unmined.rs

```diff
@@ -271,4 +297,45 @@ impl VerifiedUnminedTx {
             miner_fee,
         }
     }
+
+    /// The cost in bytes of the transaction, as defined in [ZIP-401].
+    ///
+    /// A reflection of the work done by the network in processing them (proof
+    /// and signature verification; networking overheads; size of in-memory data
+    /// structures).
+    ///
+    /// > Each transaction has a cost, which is an integer defined as:
+    /// >
+    /// >     max(serialized transaction size in bytes, 4000)
```

Comment:
rustdoc thinks this is a doctest, not sure if there's a better way to fix this

```suggestion
    /// > max(serialized transaction size in bytes, 4000)
```

Reply:
Using a block quote and "text" tells Rust that it's text, not code:
https://doc.rust-lang.org/rustdoc/documentation-tests.html#attributes